packages/projection/src/operators/Mappers/withCIP67.ts

```diff
@@ -38,11 +38,28 @@ export const withCIP67 = unifiedProjectorOperator<WithUtxo, WithCIP67>((evt) =>
       })
     )
     .filter(isNotNil);
+
+  const mintedCip67Assets = [...(evt.block.body[0].body.mint?.keys() ?? [])]
```

Comment:
Asset name and policy id is already parsed in `withMint`: it would be better to add a dependency on that mapper instead of re-parsing. You can do it like that:
```
export const withCIP67 = unifiedProjectorOperator<WithUtxo & WithMint, WithCIP67>((evt) => {
```
and
```
// prepareTypeormProjection.ts
const mapperInterDependencies: Partial<Record<MapperName, MapperName[]>> = {
  withCIP67: ['withUtxo', 'filterUtxo', 'withMint', 'filterMint'],
```

Reply:
addressed: 55fa7474e813b502fc2c81d9df6043f4a83d575a

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/993#discussion_r1402174184